native/acter/src/api/push/notification_item.rs

```diff
@@ -548,22 +571,32 @@ impl NotificationItemBuilder {
                     builder.title("Reference".to_owned())
                 }
             }
-            ActivityContent::TitleChange { new_title, .. } => builder.title(new_title.clone()),
-            ActivityContent::EventDateChange { new_date, .. } => {
-                builder.title(new_date.to_rfc3339())
+            ActivityContent::TitleChange { content, .. } => builder.title(content.new_val()),
+            ActivityContent::EventDateChange { content, .. } => {
+                if content.start_change().is_some() {
+                    let utc_start = content
+                        .start_new_val()
+                        .expect("utc start should be available");
```

Comment:
avoiding expect here, too, would be good. don't set it, if we don't have it, but don't panic...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2740/commits/5c725035d298f6fb64369163c56d3fcb3f6632b1